.travis.yml

```diff
@@ -1,18 +1,33 @@
-language: php
-
-php:
-  - 7.2
-  - nightly
-
 matrix:
+  include:
+    - language: php
+      php: 7.2
+      os: linux
+      before_script:
+        - composer install
+      after_script:
+        - sh .travis.coverage.sh
+      env:
+        - COVERAGE_FLAGS="--coverage-text --coverage-clover=clover.xml"
+    - language: php
+      php: nightly
+      before_script:
+        - composer install
+    - name: "PHP: 7.2"
+      language: sh
+      os: windows
+      before_script:
+        - choco install php --version 7.2.15 --package-parameters='"/InstallDir:C:\php"'
```

Comment:
It would be nice to fetch latest available version of 7.2.x here to download it automatically.

Reply:
Puh don't know if that is possible. I will look into it

Reply:
Found this issue:

https://github.com/chocolatey/choco/issues/800